src/Lykke.Service.PayInternal.Services/CalculationService.cs

```diff
@@ -38,15 +38,15 @@ public class CalculationService : ICalculationService
         {
             var rate = await GetRateAsync(baseAssetId, quotingAssetId, requestMarkup.Percent, requestMarkup.Pips, merchantMarkup);
 
-            await _log.WriteInfoAsync(nameof(CalculationService), nameof(GetAmountAsync), new
+            _log.Info("Rate calculation", new
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.